From 717fb4ec0a18989bf640f5603f9905a9afe4842a Mon Sep 17 00:00:00 2001
From: Justin Maggard <jmaggard@netgear.com>
Date: Fri, 21 Jul 2017 15:11:31 -0700
Subject: [PATCH 16/16] nmbd: Increase log level for common message

Silence common "Multiple responses received for a query" messages at
quiet log levels.
---

diff --git a/source3/nmbd/nmbd_namequery.c b/source3/nmbd/nmbd_namequery.c
index ae50a232987..479fbf5d332 100644
--- a/source3/nmbd/nmbd_namequery.c
+++ b/source3/nmbd/nmbd_namequery.c
@@ -106,7 +106,7 @@ static void query_name_response( struct subnet_record   *subrec,
 		}
 	} else if( rrec->num_msgs > 1) {
 
-		if( DEBUGLVL( 0 ) ) {
+		if( DEBUGLVL( 6 ) ) {
 			if (nmb->answers)
 				putip( (char *)&answer_ip, &nmb->answers->rdata[2] );
 			dbgtext( "query_name_response: " );
-- 
2.15.1

